src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -668,7 +668,7 @@ namespace LowTempRadiantSystem {
 
             thisRadSys.HotThrottlRange = Numbers(7);
             if (thisRadSys.HotThrottlRange < MinThrottlingRange) {
-                ShowWarningError("ZoneHVAC:LowTemperatureRadiant:VariableFlow: Heating throttling range too small, reset to 0.5");
+                ShowWarningError("ZoneHVAC:LowTemperatureRadiant:VariableFlow: Heating throttling range too small, reset to 0");
```

Comment:
I'm not convinced this needs to be here.  If I'm reading it right, these minimum throttling range blocks should just be removed entirely.